src/components/Header/index.tsx

```diff
@@ -42,7 +45,8 @@ export const Header: React.FC<HeaderProps> = ({
       </div>
       <div
         className="item-container right-items"
-      >
+      > 
+        <LanguageSelect {...i18n} />
```

Comment:
`i18n` can be directly imported in the `LanguageSelect` instead of passing it as a prop.